DOC-4393: Update the People documentation (#233)
exoplatform/exo-documentation: docs/PersonalApplications.rst

 * :ref:`Accepting/Denying a connection request <Accept-Deny-Connection-Request>`
 * :ref:`Disconnecting from your contacts <Disconnect-Contact>`

> To build your connections, you first need to click the humberger menu on

Reviewer: ```suggestion
To build your connections, you first need to click the hamburger menu on
```

the side navigation bar and select **People**.

|PEOPLE1|

The **People** page appears.

    .. note:: If there are so many connections, the Show More bar will appear at the page bottom. Click Show More to see more connections.
             If you don't have any connection, a simple text is displated *NO PEOPLE*.

> .. _People.Page:

Reviewer: You changed the section's ID, it may be called in other sections, so either you should restore the old id or change it everywhere called

Author: Changed.

> This People page includes the following sections:

Reviewer: ```suggestion
The People page includes the following sections:
```

|PEOPLE2|

- |image1| This section contains the following options: the number of connection displayed, 
   Filter connection by name, position or skills and a pulldown to filter the list  : Everyone or My Connections.  

 .. note:: -  *Everyone*: lists users who have registered in the eXo Platform system.

>            -  *My Connections*: lists users who have established connections with you. You can remove these connections by clicking Remove Connection.

Reviewer: ```suggestion
           -  *My Connections*: lists users connected with you. You can remove a connection by clicking on "Remove Connection" button.
```

- |image2| Connections in which you can see information like Full Name, avatar, cover image, job title
   and connection button.    

 .. note:: You can click on the information icon of the user's card which flips to display the 

>    user's about me section and its number of connections and spaces.

Reviewer: ```suggestion
   user's "about me" section and the number of connections and spaces.
```

>     .. note:: For the number of spaces, the number displayed is for all spaces that this user is a manager or member.

Reviewer: ```suggestion
    .. note:: For spaces number, it counts spaces where this user is manager or member.
```

>               If the About me description is very long, an ellipsis is displayed and the full about me text will be displayed in a tooltip. 

Reviewer: ```suggestion
              If "About me" description is very long, an ellipsis is displayed with the full text. 
```

   |PEOPLE3| 

- |image3| A block for invitations requests and Pending requests sent.

- |image4| A suggestion list of users to connect with. 
   You can either send a request to connect or decline the suggestion.   

- |image5| Leaderboards display a snapshot of users with most points. 

>       .. note:: For more information about the *Leadebroard*, you can take a look on Gamification :ref:`Leaderboard <Leaderboard>`.

Reviewer: ```suggestion
      .. note:: For more information about the *Leaderboard*, you can take a look at :ref:`Leaderboard <Leaderboard>` part on Gamification chapter .
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~

After :ref:`specifying your desired contact <SearchingForContact>`,

> you can send a connection request via one of three ways.

Reviewer: ```suggestion
you can send a connection request via one of these three ways.
```

**The first way**

Access to the People page and click on **Cancel Request** in the users' card to revoke the connection request.

|PEOPLE6|

**The second way**

1. Select the Requests Pending drawer in Requests section.

> 2. Click **Cancel Request** icon button next to the contact name, or click directly on Cancel request button in the users' card to revoke the connection request.

Reviewer: ```suggestion
2. Click **Cancel Request** icon button near the contact name or click directly on the "Cancel request" button on the users' card to revoke the connection request.
```

**The first way**

Use the Invitations application on the Welcome Back block on the Digital Workplace homepage. See :ref:`here <DW-Widgets>` for more details.

**The second way**

1. Select Invitation in the Invitations block in the My Connections page and click on it, a drawer containing all received invitations is displayed.

|PEOPLE8|

> 2. Click Confirm/ Ignore icons next to the contact name to accept/deny the request respectively.

Reviewer: ```suggestion
2. Click Confirm/ Ignore icons near the contact name to accept/deny the request respectively.
```

>     .. note:: You can also accept the request directly in the users' card and by hovering the cursor over the contact name, then click Confirm.

Reviewer: ```suggestion
    .. note:: You can also accept the request directly in the user's card by hovering the cursor over the contact name, then clicking "Confirm" button.
```
